DrawerKit/DrawerKit/Internal API/AnimationController.swift

```diff
@@ -4,9 +4,17 @@ final class AnimationController: NSObject {
     private let isPresentation: Bool
     private let configuration: DrawerConfiguration
 
-    init(isPresentation: Bool, configuration: DrawerConfiguration) {
+    let presentingDrawerAnimationActions: DrawerAnimationActions
+    let presentedDrawerAnimationActions: DrawerAnimationActions
+
+    init(isPresentation: Bool,
```

Comment:
What does `isPresentation` refer to?

Reply:
@diegopetrucci `isPresentation == true` if the drawer is being presented (i.e., going up from collapsed) and `isPresentation == false` if the drawer is being dismissed (i.e., going down to collapsed)

Reply:
Would prefer to be an enum to document this. `TransitionStatus = isPresenting | isDismissing`

Reply:
@RuiAAPeres It's a boolean. It has only two values and it's already self-documented by having a good name. I don't think creating an enumeration for it buys us any more understanding, but it adds more clutter.

Reply:
I don't really have any strong opinion on this flag vs enum. Maybe just calling it `isPresenting`.